lambdas/llama-2-13b-chat-llm/template.yml

```diff
@@ -0,0 +1,112 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: QnABot on AWS LLM Plugin for Llama-2-13b-chat
+
+Parameters:
+
+  LLMModel:
```

Comment:
Is this meant to me the SageMaker endpoint name?  If se, let's rename it to `SageMakerEndpointName` and provide more complete description.
And also add one for Arn, so you can scope the invoke policy properly.

Reply:
Addressed in the 2nd commit.